apps/backend/prisma/schema.prisma

```diff
@@ -13,7 +13,68 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum Priority {
+  LOW
+  MEDIUM
+  HIGH
+}
+
+enum Status {
+  OPEN
+  IN_PROGRESS
+  DONE
+}
+
+enum Role {
+  ADMIN
+  USER
+}
+
 model User {
-  authSchId String @id
-  email     String @unique
+  id             Int     @id @default(autoincrement())
+  authSchId      String //Ez majd a belepeshez kell
+  email          String  @unique
+  profilePicture String?
+  groups         GroupMembers[]
+  events         Event[]
+}
+
+model Group {
+  id          Int     @id @default(autoincrement())
+  name        String
+  description String?
+  members     GroupMembers[]
+  events      Event[]
+}
+
+model GroupMembers {
+  groupId Int
+  userId  Int
+  role    Role @default(USER)
+
+  @@id([groupId, userId])
+  Group   Group @relation(fields: [groupId], references: [id])
+  User    User  @relation(fields: [userId], references: [id])
+}
+
+model Event {
```

Comment:
láttam volt location is az elején, az miért került ki? szerintem az nice ha van egy opt ilyen mezo pl zh-knál

Reply:
Mivel nem volt a tablan, de amugy tenyleg van letjogosultsaga